scripts/configure_paths.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [[ ${ARCH} != "X64" && ! ( ${OS} == "macOS" && ${ARCH} == "ARM64" ) ]]; then 
+if [[ ${ARCH} != "X64" && ! ( (${OS} == "macOS" && ${ARCH} == "ARM64") || (${OS} == "Linux" && ${ARCH} == "ARM64") ) ]]; then 
```

Comment:
A minor suggestion for simplification (untested)?

```suggestion
if [[ ${ARCH} != "X64" && ! (${ARCH} == "ARM64" && (${OS} == "macOS" || ${OS} == "Linux")) ]]; then 
```
WDYT?

Reply:
I think I am missing something, you marked the conversation as resolved, and you kept the original version. Is my suggestion incorrect?

Reply:
I committed your suggestion, I might have lost it in a force push while rebasing to have a single commit. sorry